lexfloatclient.go

```diff
@@ -322,6 +326,25 @@ func HasFloatingLicense() int {
 	return int(status)
 }
 
+/*
+   FUNCTION: GetFloatinglicenseMode()
+
+   PURPOSE: Gets the mode of the floating license (online or offline).
+
+   PARAMETERS:
+   * modePtr - pointer to a buffer that receives the value of the string
+   * length - size of the buffer pointed to by the value parameter
+
+   RETURN CODES: LF_OK, LF_E_PRODUCT_ID, LF_E_NO_LICENSE, LF_E_BUFFER_SIZE
+*/
+
+func GetFloatingLicenseMode(modePtr *string) int {
```

Comment:
@muneebkq Extra line between the docs and function

Reply:
done